terra-batch-libchart/templates/_labels.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "terra-batch-libchart-coa-helm.labels" -}}
+aadpodidbinding: {{ .Values.identity.name }}
```

Comment:
I don't think `aadpodidbinding` should be in terra-batch-libchart, since it's Azure-specific (and the library chart supports both clouds)

Reply:
Updated!